content/docs/0.8.x/monitoring/prometheus/install/index.md

```diff
@@ -13,33 +13,72 @@ In order to evaluate the quality gates and allow self-healing in production, we
 
 ## Setup Prometheus
 
-After creating a project and service, you can setup Prometheus monitoring and configure scrape jobs using the Keptn CLI. Therefore you need to deploy the *prometheus-service*. 
+After creating a project and service, you can set up Prometheus monitoring and configure scrape jobs using the Keptn CLI.
 
-* Specify the version of the prometheus-service you want to deploy. Please see the [compatibility matrix](https://github.com/keptn-contrib/prometheus-service#compatibility-matrix) of the prometheus-service to pick the version that works with your Keptn.  
+Keptn doesn't install or manage Prometheus and its components. Users need to install Prometheus and Prometheus Alert manager as a prerequisite.
```

Comment:
Please move this line to the `Prerequisites` above.

Reply:
Done